locale/en/docs/guides/debugging_getting_started.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Debugging - Getting Started
+layout: docs.hbs
+---
+
+# Debugging Guide
+
+The goal of this guide is to provide you with enough information to get started
+debugging your Node.js apps and scripts.
+
+## Enable Debugging
+
+### --debug
+
+When started with the **--debug** or **--debug-brk** switches, Node.js listens
+for debugging commands defined by the [V8 Debugging Protocol][] on a TCP port,
+by default `5858`. Any debugger client which speaks this protocol can connect
+to and debug the running process.
+
+A running Node.js process can also be instructed to start listening for
+debugging messages by signaling it with `SIGUSR1` (on Linux and OS X).
```

Comment:
Since this document will appear in the version-independent "Guides" section of the doc, makes sense for it to discuss both `--debug` and `--inspect`.  But they should be in separate sections I think - inspect first, then debug. You never use both at the same time (I assume!), so having them separated seems right.  Each section should also mention which node versions support that type of debug.